packages/mdc-textfield/_variables.scss

```diff
@@ -49,3 +49,5 @@ $mdc-textarea-light-background: rgba(white, 1);
 $mdc-textarea-dark-background: rgba(48, 48, 48, 1);
 $mdc-textarea-disabled-background-on-light: rgba(249, 249, 249, 1);
 $mdc-textarea-disabled-background-on-dark: rgba(47, 47, 47, 1);
+
+$mdc-text-field-border-radius: 4px !default;
```

Comment:
My only comment RE this is we might want to go back to other things we've mixin-ified and evaluate whether we want to do this for any parameters in those components.